src/backend/expungeservice/crawler/models/case.py

```diff
@@ -12,4 +12,13 @@ def __init__(self, info, case_number, citation_number, date_location, type_statu
         self.date = datetime.date(datetime.strptime(date, '%m/%d/%Y'))
         self.violation_type, self.current_status = type_status
         self.charges = charges
+        self.balance_due_in_cents = 0
         self.case_detail_link = case_detail_link
+
+    def set_balance_due_dollar_amount(self, balance_due_dollar_amount):
+        if type(balance_due_dollar_amount) == str:
+            balance_due_dollar_amount = float(balance_due_dollar_amount.replace(',',''))
+        self.balance_due_in_cents = int(balance_due_dollar_amount * 100)
```

Comment:
Question: If line 19 and line 21 are deleted do all the tests still pass?

Reply:
Not sure, but I think you're referring to lines 19 and 20? The test case for setting nonzero balance happens to be over 1000 and has a comma in the string value, so without checking for this then the math operation (and test) does fail.

Reply:
haha I shouldn't review PR's late at night. This is perfect!